src/main/java/com/powsybl/openloadflow/network/LfNetworkLoader.java

```diff
@@ -7,6 +7,7 @@
 package com.powsybl.openloadflow.network;
 
 import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.openloadflow.network.impl.LfTopoConfig;
```

Comment:
We have a dependency in the wring direction, we maybe should move LfTopoConfig in the network package